File: ios/MapPing/Classes/Domain/Model/Part.swift
//
//  Part.swift
//  MapPing
//
//  Copyright © 2018 Mirego. All rights reserved.
//

struct Part: Codable {
    enum CodingKeys: String, CodingKey {
        case name
        case component
        case notes
        case type
        case latitude = "lat"
        case longitude = "lon"
        case address
    }

    let name: String
    let component: String
    let notes: String
    let type: String
    let latitude: Double
    let longitude: Double
    let address: String
}

File: android/app/src/main/java/com/mirego/csmapapplication/model/Part.kt
package com.mirego.csmapapplication.model

import com.google.gson.annotations.SerializedName

class Part {
    val name: String? = null

    val component: String? = null

    val notes: String? = null

    @SerializedName("lat")
    val latitude: Double? = null

    @SerializedName("lon")
    val longitude: Double? = null

    val address: String? = null
}